src/Utils/cell_advection_timescale.jl

```diff
@@ -7,9 +7,9 @@ function cell_advection_timescale(u, v, w, grid)
     vmax = maximum(abs, v)
     wmax = maximum(abs, w)
 
-    Δx = min_Δx(grid)
-    Δy = min_Δy(grid)
-    Δz = min_Δz(grid)
+    Δx = minimum_spacing(:x, (Face, Face, Face), grid)
+    Δy = minimum_spacing(:y, (Face, Face, Face), grid)
+    Δz = minimum_spacing(:z, (Face, Face, Face), grid)
```

Comment:
should we be more accurate here using 
```suggestion
    Δx = minimum_spacing(:x, (Face, Center, Center), grid)
    Δy = minimum_spacing(:y, (Center, Face, Center), grid)
    Δz = minimum_spacing(:z, (Center, Center, Face), grid)
```
?